shell.nix

```diff
@@ -143,7 +143,7 @@ let
 
 in
 haskell.project.shellFor {
-  nativeBuildInputs = nixpkgsInputs ++ localInputs ++ [ sphinxTools ];
+  #nativeBuildInputs = nixpkgsInputs ++ localInputs ++ [ sphinxTools ];
```

Comment:
I was getting `error: nodejs-12_x has been removed` errors when entering the shell.
But the term `nodejs-12_x` does not appear in `plutus-apps` 
Uncommenting this line fixes it, so I'll try and narrow it down until I find the broken derivation.